docs/server/source/http-client-server-api.rst

```diff
@@ -154,7 +154,15 @@ Transactions
    is in a committed block.
 
    .. note::
-   
+       In the async and sync modes, after a successful HTTP response is returned, the transaction may still be rejected later on. All the transactions are recorded internally before the HTTP response is returned. Nevertheless, the following should be noted:
+
+       - Recorded transactions including the failed ones are not exposed in any of the BigchainDB or Tendermint APIs.
```

Comment:
Surely there must be _some_ Tendermint API to get the block at a given height, including its list of transactions? What about this one:

https://tendermint.github.io/slate/?shell#block

Reply:
I am referring to WAL records here. I do want to mention that they are recorded but do not want to provide any details or use the WAL term since those are not well documented and can change any time. Perhaps, you can think about a better way to call those transactions?

Reply:
Oh, sorry, I got confused by the phrase "recorded transactions" thinking it meant something more broad. I think it's probably okay to mention the WAL (but spell it out), maybe with a note that Tendermint might change how tat works in the future.